app/src/main/java/com/zeapo/pwdstore/ClipboardService.kt

```diff
@@ -85,11 +83,21 @@ class ClipboardService : Service() {
 
     private fun clearClipboard() {
         val deepClear = settings.getBoolean("clear_clipboard_20x", false)
-        val clipboardManager = getSystemService<ClipboardManager>()
+        val clipboard = getSystemService<ClipboardManager>()
 
-        if (clipboardManager is ClipboardManager) {
+        if (clipboard is ClipboardManager) {
             scope.launch {
```

Comment:
Not entirely sure how relevant this is, but having this scoped according to the lifecycle of the activity may mean it is cancelled if the user exits the activity. Would it be safer to use GlobalScope here?

Reply:
This is scoped to the service, I don't think an activity lifecycle is involved?

Reply:
True, I missed that this is in a service and not an activity.